src/libsync/ownsql.cpp

```diff
@@ -303,6 +303,10 @@ void SqlQuery::bindValue(int pos, const QVariant& value)
                 res = sqlite3_bind_null(_stmt, pos);
             }
             break; }
+        case QVariant::ByteArray: {
+            auto ba = value.toByteArray();
+            res = sqlite3_bind_text(_stmt, pos, ba.constData(), ba.size(), SQLITE_TRANSIENT);
```

Comment:
And this is just an optimisation, maybe i could leave it out and then it would use the text16 fallback as toString which is just a bit of a waste.